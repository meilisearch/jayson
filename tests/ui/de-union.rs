use jayson::Jayson;

#[derive(Jayson)]
union Union {
    x: i32,
}

fn main() {}
