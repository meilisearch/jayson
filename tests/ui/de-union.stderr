error: currently only structs and enums are supported by this derive
 --> tests/ui/de-union.rs:3:10
  |
3 | #[derive(Jayson)]
  |          ^^^^^^
  |
  = note: this error originates in the derive macro `Jayson` (in Nightly builds, run with -Z macro-backtrace for more info)
